src/desktop/test/lib/admin_only.test.js

```diff
@@ -1,35 +1,31 @@
-/*
- * decaffeinate suggestions:
- * DS102: Remove unnecessary code created because of implicit returns
- * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
- */
 const _ = require("underscore")
 const sinon = require("sinon")
 const Backbone = require("backbone")
-const adminOnly = require("../../lib/admin_only")
+const { adminOnly } = require("../../lib/admin_only")
 
-describe("adminOnly middleware", function () {
-  beforeEach(function () {
-    return (this.next = sinon.stub())
+describe("adminOnly middleware", () => {
+  let next
+  let req
+
+  beforeEach(() => {
+    next = sinon.stub()
```

Comment:
This looks like it would be a pretty straightforward test to convert to jest and remove sinon.

Reply:
Definitely, test conversions weren't the original intent for this PR, I can quickly convert the tests in the `src/desktop/test/lib/` folder if you'd like.